tests/ext/sandbox/static_tracing_closures_will_not_bind_this.phpt

```diff
@@ -23,5 +23,5 @@ $foo->test();
 ?>
 --EXPECT--
 Foo::test()
-Cannot trace non-static method with static tracing closure
-Successfully triggered flush with trace of size 1
+TRACED Foo::test()
```

Comment:
So... this particular function doesn't need `$this`, but in general `$this` of the closure was bound to the object it traces, and the engine wouldn't let you bind the static closure to the object. What changed?

Reply:
Magic in zai symbols :-)
We're not asking the engine to rebind it first, but we do it instead manually, with an op_array copy:

https://github.com/DataDog/dd-trace-php/blob/8fee58ff83610b74e21631c5473fb96e19bcf14b/zend_abstract_interface/symbols/call.c#L156-L176